### ISSUE TITLE
Subscribe: Using colour alone to identify which fields are in error 

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#135]
URL: https://recalls-rappels.canada.ca/en/subscribe
User Agent: Chrome
Language: Both

Issue Description: Using colour alone to identify which fields are in error. Color alone cannot be used as the only visual means of conveying information or distinguishing a visual event. [Failure of Success Criterion 1.4.1 due to identifying required or error fields using color differences only](https://www.w3.org/WAI/WCAG21/Techniques/failures/F81)

Steps to reproduce: 
1. Navigate to the Subscribe page.
2. scroll to the bottom of the page and trigger the Submit button
3. Page reloads with error messages at the top of the page. 
4. Fields in error are displayed in red. See screenshot

![image](https://user-images.githubusercontent.com/65728700/159687355-0f5f5349-6d7c-4f2b-ab43-9314af949623.png)

Expected results: One of the recommended techniques is [G14: Ensuring that information conveyed by color differences is also available in text](https://www.w3.org/WAI/WCAG21/Techniques/general/G14)

**Comment by davidgotlieb** [comment]
See response in #105:

We are using the default WxT error handling. From OP:

WxT is not 100% WET compliant. Currently, error message handling in WxT is not done the same as it WET. It hasn't been implemented yet. It will be a major undertaking to get the WET error message handling into WxT, when it does get there, it will be pulled into RSAMS as part of the regular WXT update.

**Comment by davidgotlieb** [comment]
@mmiron-HC same as other error handling issues

We are using the default WxT error handling. From OP:

WxT is not 100% WET compliant. Currently, error message handling in WxT is not done the same as it WET. It hasn't been implemented yet. It will be a major undertaking to get the WET error message handling into WxT, when it does get there, it will be pulled into RSAMS as part of the regular WXT update.
